nevergrad/benchmark/experiments.py

```diff
@@ -1040,6 +1040,31 @@ def control_problem(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
                             yield xp
 
 
+@registry.register
+def neuro_control_problem(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """MuJoCo testbed. Learn neural policies."""
+    seedg = create_seed_generator(seed)
+    num_rollouts = 1
+    funcs = [control.Swimmer(num_rollouts=num_rollouts, intermediate_layer_dim=(50,), random_state=seed),
+             control.HalfCheetah(num_rollouts=num_rollouts, intermediate_layer_dim=(50,), random_state=seed),
+             control.Hopper(num_rollouts=num_rollouts, intermediate_layer_dim=(50,), random_state=seed),
+             control.Walker2d(num_rollouts=num_rollouts, intermediate_layer_dim=(50,), random_state=seed),
+             control.Ant(num_rollouts=num_rollouts, intermediate_layer_dim=(50,), random_state=seed),
+             control.Humanoid(num_rollouts=num_rollouts, intermediate_layer_dim=(50,), random_state=seed)
+             ]
```

Comment:
always benefinicial on the long run to avoid duplicated code ;) (easier to maintain)

```suggestion
    funcs = [
        Env(num_rollouts=num_rollouts, intermediate_layer_dim=(50,), random_state=seed) for env in [
            control.Swimmer, control.HalfCheetah, control.Hopper, control.Walker2d, control.Ant, control.Humanoid
        ]
    ]
```